README.md

```diff
@@ -39,31 +39,40 @@ $ brew update
 $ brew install carthage
```

Comment:
nits: Let's remove the entire `Carthage` section. 🙏 

cc: @wubai-appier @michaelrevlis